api_client/python/timesketch_api_client/importer.py

```diff
@@ -210,10 +362,11 @@ def add_dict(self, entry):
         if not isinstance(entry, dict):
             raise TypeError('Entry object needs to be a dict.')
 
-        if self._count >= self._threshold:
+        if self._count >= self._threshold_entry:
             self.flush(end_stream=False)
             self._reset()
 
+        self._fix_dict(entry)
```

Comment:
There is a fair amount of fixing in the method call here - can you add a comment outlining what is happening, and refer to the method for more details?

Reply:
done